ros2bag/ros2bag/verb/play.py

```diff
@@ -37,131 +37,135 @@ def positive_float(arg: str) -> float:
     return value
 
 
+def add_player_arguments(parser: ArgumentParser) -> None:
+    add_standard_reader_args(parser)
+    parser.add_argument(
+        '--read-ahead-queue-size', type=int, default=1000,
+        help='size of message queue rosbag tries to hold in memory to help deterministic '
+             'playback. Larger size will result in larger memory needs but might prevent '
+             'delay of message playback.')
+    parser.add_argument(
+        '-r', '--rate', type=check_positive_float, default=1.0,
+        help='rate at which to play back messages. Valid range > 0.0.')
+    parser.add_argument(
+        '--topics', type=lambda s: list(s.split(' ')), default=[], metavar='topic',
+        help='Space-delimited list of topics to play.')
+    parser.add_argument(
+        '--services', type=lambda s: list(s.split(' ')), default=[], metavar='service',
+        help='Space-delimited list of services to play.')
+    parser.add_argument(
+        '-e', '--regex', default='',
+        help='Play only topics and services matches with regular expression.')
+    parser.add_argument(
+        '-x', '--exclude-regex', default='',
+        help='regular expressions to exclude topics and services from replay.')
+    parser.add_argument(
+        '--exclude-topics', type=lambda s: list(s.split(' ')), default=[], metavar='topic',
+        help='Space-delimited list of topics not to play.')
+    parser.add_argument(
+        '--exclude-services', type=lambda s: list(s.split(' ')), default=[], metavar='service',
+        help='Space-delimited list of services not to play.')
+    parser.add_argument(
+        '--qos-profile-overrides-path', type=FileType('r'),
+        help='Path to a yaml file defining overrides of the QoS profile for specific topics.')
+    parser.add_argument(
+        '-l', '--loop', action='store_true',
+        help='enables loop playback when playing a bagfile: it starts back at the beginning '
+             'on reaching the end and plays indefinitely.')
+    parser.add_argument(
+        '--remap', '-m', default='', nargs='+',
+        help='list of topics to be remapped: in the form '
+             '"old_topic1:=new_topic1 old_topic2:=new_topic2 etc." ')
+    parser.add_argument(
+        '--storage-config-file', type=FileType('r'),
+        help='Path to a yaml file defining storage specific configurations. '
+             'See storage plugin documentation for the format of this file.')
+    clock_args_group = parser.add_mutually_exclusive_group()
+    clock_args_group.add_argument(
+        '--clock', type=positive_float, nargs='?', const=40, default=0,
+        help='Publish to /clock at a specific frequency in Hz, to act as a ROS Time Source. '
+             'Value must be positive. Defaults to not publishing.')
+    clock_args_group.add_argument(
+        '--clock-topics', type=lambda s: list(s.split(' ')), default=[],
+        help='List of topics separated by spaces that will trigger a /clock update '
```

Comment:
As fujitatomoya mentioned, "and quoted" should also be added to the description.